cmd/simulateEvol/nonCoding.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+	"math/rand"
+	"os"
+)
+
+// NonCodingSettings defines usage settings for the simulateEvol nonCoding subcommand.
+type NonCodingSettings struct {
+	TreeFile               string
+	FastaFile              string
+	OutFile                string
+	UnitBranchLength       float64
+	SubstitutionMatrixFile string
+	NumNodes               int
+	GammaAlpha             float64
+	GammaBeta              float64
+	GcContent              float64
+	LenSeq                 int
+	SetSeed                int64
+	NewickOut              string
+}
+
+// NonCodingUsage defines the usage statement for the simulateEvol nonCoding subcommand.
```

Comment:
NonCoding  emulates the Jukes Cantor model, by simulating evolution, but doesn't Genic do the something similar (?) by first reading a tree and then simulating evolution, then extracting fasta information from tree nodes?